spec/services/exports/audience_export_service_spec.rb

```diff
@@ -4,14 +4,65 @@
 
 describe Exports::AudienceExportService do
```

Comment:
Could we add two more tests please:

1. When none of the options is set
2. When all of the options are set